### PR TITLE
Add pascals-triangle exercise

### PR DESCRIPTION
**Pull request by dalexj**

### CODE REVIEW

File: exercises/pascals-triangle/pascals_triangle_test.exs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("allergies.exs")
```
Comment (rubysolo) [inline exercises/pascals-triangle/pascals_triangle_test.exs]: Please fix the filename.

Comment (dalexj) [inline exercises/pascals-triangle/pascals_triangle_test.exs]: thanks, good catch

File: exercises/pascals-triangle/pascals_triangle_test.exs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PascalsTriangleTest do
+  use ExUnit.Case
+
+  test "test one row" do
+    assert [[1]] == PascalsTriangle.rows(1)
```
Comment (parkerl) [inline exercises/pascals-triangle/pascals_triangle_test.exs]: Can you please flip these so actual is on the left?

File: exercises/pascals-triangle/pascals_triangle_test.exs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+end
+
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PascalsTriangleTest do
+  use ExUnit.Case
+
+  test "test one row" do
```
Comment (parkerl) [inline exercises/pascals-triangle/pascals_triangle_test.exs]: @dalexj we've been putting a commented out `@tag pending` on the first spec as an example for submitters. I think we should be consistent.

Comment (parkerl) [inline exercises/pascals-triangle/pascals_triangle_test.exs]: Also you have `test "test..." in the names of these specs.

**Comment by parkerl** [comment]
@dalexj nice! A coulomb small issues, but it looks good!

**Comment by dalexj** [comment]
changed to match your comments @parkerl

**Comment by parkerl** [comment]
Looks good to me, @rubysolo?

**Comment by parkerl** [comment]
```
cou·lomb
ˈko͞oˌläm,-ˌlōm`
noun
the SI unit of electric charge, equal to the quantity of electricity conveyed in one second by a current of one ampere.
```

Autocorrect is just making sure you are up on your SAT words. 😠

**Pull request merged**